File: text/jython/delete/text_delete.py
#! /opt/jython/bin/jython
# -*- coding: utf-8 -*-
#
#	delete/text_delete.py
#
#					Oct/12/2016
import	sys
import	string
#
# ---------------------------------------------------------------
sys.path.append ('/var/www/data_base/common/python_common')
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_text_manipulate import text_read_proc
from jython_text_manipulate import text_write_proc
#
from text_manipulate import dict_delete_proc
# ---------------------------------------------------------------
#
sys.stderr.write ("*** 開始 ***\n")
#
file_in = sys.argv[1]
id_in = sys.argv[2]
print ("%s" % id_in)

dict_aa = text_read_proc	(file_in)

dict_bb = dict_delete_proc (dict_aa,id_in)

text_write_proc (file_in,dict_bb)
#
sys.stderr.write ("*** 終了 ***\n")
# ---------------------------------------------------------------

File: text/deno/delete/text_delete.ts
// ---------------------------------------------------------------
//	text_delete.ts
//
//					May/01/2020
//
// ---------------------------------------------------------------
import { text_read_proc } from "./text_manipulate.ts"
import { text_write_proc } from "./text_manipulate.ts"

// ---------------------------------------------------------------
console.error ("*** 開始 ***")
//
var file_txt:string = Deno.args[0]
var key:string = Deno.args[1]

console.log (file_txt)
console.log (key)

var dict_aa = text_read_proc (file_txt)

if (key in dict_aa)
	{
	delete dict_aa[key]
	text_write_proc (file_txt,dict_aa)
	}

console.error ("*** 終了 ***")

// ---------------------------------------------------------------
